src/fairseq2/models/wav2vec2/_config.py

```diff
@@ -42,6 +42,8 @@ class Wav2Vec2Config:
     from the transformer. """
 
     # Mask
+    mask_codebase: str = "fairseq2"
```

Comment:
we need to find an appropriate name for it if we keep it...

Reply:
we can create a custom Masker object now

Reply:
Can already refactored this into a new implementation https://github.com/facebookresearch/fairseq2/blob/v0.5.0a1/src/fairseq2/models/wav2vec2/masker.py. Does not need to be renamed or ported.